RBS.Auth.WebApi/RBS.Auth.WebApi/Controllers/AuthController.cs

```diff
@@ -1,46 +1,62 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Options;
 using RBS.Auth.Common;
+using RBS.Auth.Common.Models;
 using RBS.Auth.Services.Interfaces.Authenticate;
 using RBS.Auth.Services.Interfaces.Tokens;
 using RBS.Auth.WebApi.Models;
 
-namespace RBS.Auth.WebApi.Controllers
+namespace RBS.Auth.WebApi.Controllers;
+
+[Route("api/auth")]
```

Comment:
Consider to use routing attribute of the form `[Route("api/[controller]")]`.
It is more convenient way to describe routes

Reply:
Fixed